### DISCUSSION TITLE
build warning unused-parameter on omnios

### DISCUSSION THREAD

**Discussion by jbereuter** [bareos/bareos#2047] [General]
Hi, i dont know if this warning is problematical - easy (too easy) solution is adding -Wno-unused-variable to the compiler in this part of the build. I am building on omnios.

`/home/jub/bareos/bareos-master/core/src/lib/scsi_lli.cc: In function 'bool CheckScsiAtEod(int)':
/home/jub/bareos/bareos-master/core/src/lib/scsi_lli.cc:261:25: error: unused parameter 'fd' [-Werror=unused-parameter]
  261 | bool CheckScsiAtEod(int fd) { return false; }
      |                     ~~~~^~
cc1plus: all warnings being treated as errors
*** Error code 1``

**Comment by arogge** [comment]
If you don't give the parameter a name, the compiler understands that you have no intention to use it. So you can fix this by changing it to:
```
bool CheckScsiAtEod(int) { return false; }
```

As we currently don't do integration builds for Solaris and derivatives anymore, this was probably overlooked during refactoring. Feel free to open a PR to fix this (and other build issues you run into) - we'll happily accept that.
